plugins/chat/assets/javascripts/discourse/components/chat/thread/list.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="chat-thread-list">
+  {{#if @includeHeader}}
+    <div class="chat-thread__header">
```

Comment:
Just copied this from the chat single thread component for now -- we could convert this into a shared component, but more likely the index header will be different so let's just leave it as is for now